### ISSUE TITLE
2FA failed with mobile TAN

### ISSUE THREAD

**Issue by Clint86** [andywuest/harbour-commbank#1]
The 2FA authentification failed after entering the mobile tan.
I got no feedback whats wrong.

How do I have to enter the client_id? I tried with User_... in front.

Can you activate the comment function in your telegram channel to comment and feedback?

**Comment by andywuest** [comment]
Yes - client_id starts with "User_" prefix - and you have to add it like that.
When you get an sms you passed the first validation round. 

can you start the app from the command line - then you will get all the request and responses on the console. Then you have to send me latest response that most likely contains an error or make an error in the qml page. 

> Can you activate the comment function in your telegram channel to comment and feedback?
should work now - please try - sorry. is the first channel i created.

**Comment by andywuest** [comment]
Create a Group now - please try with this link : https://t.me/+HvQpakSmIKYxZmQy
